modules/mono/glue/Managed/Files/StringExtensions.cs

```diff
@@ -928,7 +928,7 @@ public static float[] SplitFloats(this string instance, string divisor, bool all
 
             while (true)
             {
-                int end = instance.Find(divisor, from);
+                int end = instance.Find(divisor);
```

Comment:
How does this work without `from` if there are several divisors?

Reply:
Well, I don't know for sure, but I think it doesn't. Still, this code didn't work before either...

I didn't know if I should try to refactor so much code for what was originally supposed to be just a bug fix, but I just looked it up and it seems IndexOf does actually have this functionality... brb.